CTFd/templates/teams.html

```diff
@@ -22,8 +22,8 @@
         <tbody>
         {% for team in teams %}
             <tr>
-                <td><a href="/team/{{ team.id }}">{{ team.name }}</a></td>
-                <td>{% if team.website and team.website.startswith('http://') %}<a href="{{ team.website }}">{{ team.website }}</a>{% endif %}</td>
+                <td><a href="{{ request.script_root }}/team/{{ team.id }}">{{ team.name }}</a></td>
+                <td>{% if team.website and team.website.startswith('http://') %}<a href="{{ request.script_root }}{{ team.website }}">{{ team.website }}</a>{% endif %}</td>
```

Comment:
This should not have a `{{ request.script_root }}` as it's a user provided website.